src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1542,6 +1542,10 @@ internal void SetupWin32Resources(CommonPEModuleBuilder moduleBeingBuilt, Stream
             Machine machine;
             switch (platform)
             {
+                case Platform.Arm64:
+                    machine = (Machine)0xAA64;
+                    break;
```

Comment:
Can you please open an issue somewhere to add this to System.Reflection.Metadata.Machine? Perhaps [https://github.com/dotnet/roslyn/issues/25185](https://github.com/dotnet/roslyn/issues/25185) ? Also, please add a comment (or open an issue) to fix this line when that has been done.

Reply:
Done. #25185 is the issue I'm using to track that.